tasks/sudodoki/2016-05-19-task-9/vector-bfs-dfs.js

```diff
@@ -0,0 +1,27 @@
+var tree = ['A', 'B', 'C', undefined, 'D', 'E', 'F', 'G', undefined, undefined, undefined, undefined, undefined, 'H'];
+
+var BRANCHING_FACTOR = 3;
+
+// TODO: dynamically add index calculation functions
+const hleft = (i) => (i + 1) * BRANCHING_FACTOR - 2
```

Comment:
would probably do using `getChild(level, order, branchingFactor)` + loop over index from 0 to branchingFactor